### PR TITLE
Fix devnet tests

### PR DESCRIPTION
**Pull request by gigimon**

### CODE REVIEW

**Comment by gigimon** [comment]
@kristinaNikolaevaa https://github.com/neonlabsorg/neon-tests/actions/runs/8048176305

File: integration/tests/economy/test_economics.py [hunk 1]
```diff
@@ -765,6 +765,7 @@ def test_deploy_big_contract_with_structures(self, client_and_price, account_wit
         )
         get_gas_used_percent(w3_client, receipt)
 
+    @pytest.mark.skip(reason="https://neonlabs.atlassian.net/browse/NDEV-2136")
```
Comment (kristinaNikolaevaa) [inline integration/tests/economy/test_economics.py]: it is fixed

File: integration/tests/basic/rpc/test_rpc_estimate_gas.py [hunk 2]
```diff
@@ -108,7 +108,7 @@ def test_rpc_estimate_gas_erc20(self, erc20_simple):
 
         assert "gas" in transaction
         estimated_gas = transaction["gas"]
-        assert estimated_gas == 1_415_040
+        assert estimated_gas == 1_394_160
```
Comment (kristinaNikolaevaa) [inline integration/tests/basic/rpc/test_rpc_estimate_gas.py]: for latest versions 1_415_040 value is actual

**Review by kristinaNikolaevaa — CHANGES_REQUESTED** [review]

File: integration/tests/basic/rpc/test_rpc_estimate_gas.py (hunk 2 above)
Comment (gigimon) [reply integration/tests/basic/rpc/test_rpc_estimate_gas.py]: @kristinaNikolaevaa fixed for devnet and stands :)

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**